guides/source/layout.html.erb

```diff
@@ -20,9 +20,9 @@
   <meta property="og:type" content="website" />
 </head>
 <body class="guide">
-  <% if @edge %>
+  <% if badge_version = @edge ? "edge" : @version[/v\d+.\d+/]  %>
```

Comment:
Think we can simply an just do this, and allow any version to pass through, etc for testing:

```suggestion
  <% if badge_version = @edge ? "edge" : @version %>
```

Reply:
👍

Reply:
@vipulnsward Sure, it would make the badge look like this:
<img width="884" alt="image" src="https://user-images.githubusercontent.com/28561/98586710-60a30780-22c9-11eb-9aa0-4ef3c87363b1.png">